topics/earth-system/tutorials/ocean-data-view/tutorial.md

```diff
@@ -0,0 +1,250 @@
+{: .details}
+
+# Ocean Data View
+## Map your data
+
+> <tip-title>Copy pasting between computer and ODV</tip-title>
+> You can expand the ODV left panel (where there are 3 dots, vertically) to access the "clipboard" menu, and paste the content you want to paste on a ODV form. From there you can copy paste everything from one side to the other. Then, click outside of this panel to collapse it.
+> 
+> ![Image showing in transparent on the left of the ODV interface the clipboard](../../images/coastal_water_dyn/clipboard.png)
+{: .tip}
+
+> <tip-title>ODV - Disconnected</tip-title>
+> If at one point your ODV interface becomes grey with a red pannel "X ODV - Disconnected", do NOT panik ;) you just need to reload your tab (circular arrow top left)
+{: .tip}
+
+> <hands-on-title>Loading Argo data</hands-on-title>
+>
+> 1. Go to the top left of ODV and click on "Open..."
+> 2. Click on "1900392_prof.nc" of the pop-up screen and on  **Open**
+> 3. Press **Next >** for "Select Dimensions (Step 1 of 4)"
+> 4. Press **Next >** for "Associate Meta Variables (Step 2 of 4)"
+> 5. Press **Next >** for "Select Primary Variable (Step 3 of 4)"
+> 6. Press **Finish** for "Subset Dimensions (Step 4 of 4)"
+> 7. "Enter netCDF Setup Name:" write "argo_prof_data", then, press **OK**
+{: .hands_on}
+![Map of all the stations of the Argo glider](../../images/coastal_water_dyn/argo_stations.png)
+
+> <question-title></question-title>
+>
+> 1. What are the longitude and latitude of the red dot ?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. On the to right window you can read Longitude 7°E and Latitude 21°S.
+> >
+> {: .solution}
+>
+{: .question}
+
+## Create profiles curves
+
+> <hands-on-title>Create a multiple windows view</hands-on-title>
+> 1. On the top left press **+**
+> 2. Click on **2 Station Windows**
+> ![Image showing where to go to have the 2 STATION Windows](../../images/coastal_water_dyn/2_station.png)
+> 3. Two more slots appear follow the indication and press + on your keyboard (or SHIFT and +).
+>
+> Here you have created a multiple view on ODV
+{: .hands_on}
+
+> <hands-on-title>Create profiles</hands-on-title>
+> 1. On the biggest window click on the right side of your mousse
+> 2. Select **X-Variable**, press **TEMP [degree_Celsius]** and OK.
+> ![Image of the selection of the pression](../../images/coastal_water_dyn/xvar_temp.png)
+> 3. Select **Y-Variable**, press **PRES [decibar]** and OK. Your profile is now uspide down. 
+> 4. To correct that click right again go on **Y-Variable** and on the bottom of the pop-up screen check the box "Reverse range" and OK.
+> ![ODV interface with the map and the 1st profile pression temperature](../../images/coastal_water_dyn/1st_profile.png)
+> 5. For the smallest window do the same but for the abscisse select **TEMP_ADJUSTED [degree_Celsius]**. This can be usefull to compare the 2 curves.
+> ![Resulting image of the 2 windows configurated](../../images/coastal_water_dyn/2_profiles.png)
```

Comment:
```suggestion
>    ![Image of the selection of the pression](../../images/coastal_water_dyn/xvar_temp.png)
> 3. Select **Y-Variable**, press **PRES [decibar]** and OK. Your profile is now uspide down. 
> 4. To correct that click right again go on **Y-Variable** and on the bottom of the pop-up screen check the box "Reverse range" and OK.
>    ![ODV interface with the map and the 1st profile pression temperature](../../images/coastal_water_dyn/1st_profile.png)
> 5. For the smallest window do the same but for the abscisse select **TEMP_ADJUSTED [degree_Celsius]**. This can be usefull to compare the 2 curves.
>    ![Resulting image of the 2 windows configurated](../../images/coastal_water_dyn/2_profiles.png)
```